tests/api.v2/controllers/plotController.test.js

```diff
@@ -0,0 +1,68 @@
+// @ts-nocheck
+const plotController = require('../../../src/api.v2/controllers/plotController');
+const Plot = require('../../../src/api.v2/model/Plot');
+
+const { OK } = require('../../../src/utils/responses');
+
+jest.mock('../../../src/api.v2/model/Plot');
+
+const mockRes = {
+  json: jest.fn(),
+  send: jest.fn(),
+};
+
+const mockExperimentId = 'experimentId';
+const mockPlotUuid = 'plotUuid';
+
+const plotInstance = new Plot();
+
+describe('plotController', () => {
+  beforeEach(async () => {
+    jest.clearAllMocks();
+  });
+
+  it('saving plot works correctly', async () => {
```

Comment:
do you have a test for getting plot working correctly?

Reply:
The test case for this should be named `getting plot works correctly`.